subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/catalog/AbstractVersionCatalogIntegrationTest.groovy

```diff
@@ -43,5 +43,6 @@ abstract class AbstractVersionCatalogIntegrationTest extends AbstractHttpDepende
         """
         resolve.expectDefaultConfiguration("runtime")
         resolve.prepare()
+        executer.withPluginRepositoryMirrorDisabled() // otherwise the plugin portal fixture doesn't work!
```

Comment:
Should we maybe add this method to the plugin portal fixture or have the plugin portal fixture disable the repository mirrors automatically?

Reply:
Which plugin portal fixture?